test/controllers/AdditionalMetadataNavigationControllerSpec.scala

```diff
@@ -131,12 +131,13 @@ class AdditionalMetadataNavigationControllerSpec extends FrontEndTestHelper {
       checkCommonFileNavigationElements(fileSelectionPageAsString, parentFolder, folderId, fileId, selectedFolderId)
       fileSelectionPageAsString.contains(
         s"""
-           |                <button name="pageSelected" data-prevent-double-click="true" class="govuk-button" type="submit" data-module="govuk-button" role="button" value="${currentPage + 1}">
+           |                <button name="pageSelected" data-prevent-double-click="true" class="govuk-button" type="submit """.stripMargin +
```

Comment:
So you need the string concatenation here? Can you add another new line as it's a multi line string?